src/provider/application.js

```diff
@@ -63,6 +63,18 @@ class Application extends EventEmitter {
     // Resize for slow loading images
     document.addEventListener('load', this.imageRequestResize.bind(this), true);
 
+    // Event listener and handler to set visual focus indicator (outline style) on the frame
```

Comment:
NP: You're intending to add an outline for your use, but the implementation here is general. Might save the "(outline style)" bit for elsewhere, or rephrase it as an example?

Reply:
I rephrase the comment to make it more generic